src/plugins/formActions/CreateWizard.js

```diff
@@ -54,7 +54,7 @@ export default class CreateWizard {
     addTitle(sections) {
         const row = {
             control: 'textfield',
-            cssClass: 'l-input-lg',
+            cssClass: 'l-input-lg form-title',
```

Comment:
unfortunately, these tests will need to run against build 6 and so we haven't been making these changes since that code is frozen :(